library/common/config/config.cc

```diff
@@ -389,7 +393,11 @@ R"(
     circuit_breakers: *circuit_breakers_settings
     typed_extension_protocol_options: *h1_protocol_options
   - name: base_h2
-    http2_protocol_options: {}
+    typed_extension_protocol_options:
+      envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+        "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+        explicit_http_config:
+          http2_protocol_options: {}
```

Comment:
Just to confirm - this will still configure this cluster to use h2 without protocol negotiation, correct?

Reply:
yeah, auto_config does alpn, explicit_http does whatever protocol you explicitly configure.